test/server.py

```diff
@@ -42,12 +42,14 @@ def __init__(self,
                  host='localhost',
                  ready_event=None,
                  h2=True,
-                 secure=True):
+                 secure=True,
+                 secure_auto_wrap_socket=True):
```

Comment:
Yes, I thought about this, but it would have required to replace a lot of True/False values for current tests, producing a meaninglessly (at this moment) large diff.

However, I think I've got a better idea how to make it work without this new boolean flag.  Patch is on the way 💪